src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.cs

```diff
@@ -169,7 +170,7 @@ private unsafe void Dispose(bool disposing)
             }
         }
 
-        internal static unsafe int ReadUnalignedI4(int* p)
+        private static unsafe int ReadUnalignedI4(int* p)
```

Comment:
`ResourceReader` had a few methods marked as `internal` but not actually used outside of the class. I made them `private` to hopefully avoid these contributing to new thread-safety bugs in the future.

Reply:
I assumed we also checked if this was used in System.Resources.Extesnions and their tests? That assembly also compiles this file, so it will be good to make sure.

Reply:
Confirmed that that project still builds and its tests pass. It includes RuntimeResourceSet but doesn't seem to reference it at all...